base_default_lang_translate/models/res_lang.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Le Filament (https://le-filament.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ResLang(models.Model):
+    _inherit = "res.lang"
+
+    default_lang = fields.Boolean("Default Language", default=False)
```

Comment:
A non-blocking suggestion.  `is_default_lang` may be a little more explicit.

Reply:
Makes sense, I have just updated it !